addon/components/cell.js

```diff
@@ -76,22 +55,24 @@ export default Component.extend(PropTypeMixin, {
   /**
    * Get definition for current cell
    * @param {BunsenCell} cellConfig - current cell
-   * @param {BunsenCell[]} cellDefinitions - list of cell definitions
+   * @param {Object<String, BunsenCell>} cellDefinitions - list of cell definitions
```

Comment:
Haven't seen this notation before. Does this mean you have a simple dictionary of the shape:
`{'foo': bunsenCellOne, 'bar': bunsenCellTwo, etc...}` ?